contributing/development/core_and_modules/core_types.rst

```diff
@@ -128,107 +129,137 @@ its storage strategy. Prefer ``Vector<>`` (or ``LocalVector<>``) over
+| ``Pair``               | ``std::pair``            | Stores a single key-value pair.                                                       |
++------------------------+--------------------------+---------------------------------------------------------------------------------------+
+
+References
+~~~~~~~~~~
 
 -  `core/string/ustring.h <https://github.com/godotengine/godot/blob/master/core/string/ustring.h>`__
-
-StringName
-----------
-
-StringNames are like a String, but they are unique. Creating a
-StringName from a string results in a unique internal pointer for all
-equal strings. StringNames are useful for using strings as
-identifier, as comparing them is basically comparing a pointer.
-
-Creation of a StringName (especially a new one) is slow, but comparison
-is fast.
-
-References:
-~~~~~~~~~~~
-
 -  `core/string/string_name.h <https://github.com/godotengine/godot/blob/master/core/string/string_name.h>`__
+-  `core/templates/vector.h <https://github.com/godotengine/godot/blob/master/core/templates/vector.h>`__
+-  `core/templates/local_vector.h <https://github.com/godotengine/godot/blob/master/core/templates/local_vector.h>`__
+-  `core/variant/array.h <https://github.com/godotengine/godot/blob/master/core/variant/array.h>`__
+-  `core/templates/list.h <https://github.com/godotengine/godot/blob/master/core/templates/list.h>`__
+-  `core/templates/fixed_vector.h <https://github.com/godotengine/godot/blob/master/core/templates/fixed_vector.h>`__
+-  `core/templates/span.h <https://github.com/godotengine/godot/blob/master/core/templates/span.h>`__
+-  `core/templates/hash_set.h <https://github.com/godotengine/godot/blob/master/core/templates/hash_set.h>`__
+-  `core/templates/rb_set.h <https://github.com/godotengine/godot/blob/master/core/templates/rb_set.h>`__
+-  `core/templates/vset.h <https://github.com/godotengine/godot/blob/master/core/templates/vset.h>`__
+-  `core/templates/hash_map.h <https://github.com/godotengine/godot/blob/master/core/templates/hash_map.h>`__
+-  `core/templates/a_hash_map.h <https://github.com/godotengine/godot/blob/master/core/templates/a_hash_map.h>`__
+-  `core/templates/oa_hash_map.h <https://github.com/godotengine/godot/blob/master/core/templates/oa_hash_map.h>`__
+-  `core/templates/rb_map.h <https://github.com/godotengine/godot/blob/master/core/templates/rb_map.h>`__
+-  `core/variant/dictionary.h <https://github.com/godotengine/godot/blob/master/core/variant/dictionary.h>`__
+-  `core/templates/pair.h <https://github.com/godotengine/godot/blob/master/core/templates/pair.h>`__
 
 Math types
 ----------
 
-There are several linear math types available in the core/math
+There are several linear math types available in the ``core/math``
 directory.
 
-References:
-~~~~~~~~~~~
+References
+~~~~~~~~~~
 
 -  `core/math <https://github.com/godotengine/godot/tree/master/core/math>`__
 
 NodePath
```

Comment:
Perhaps this one belongs in the table?
It's not a typical 'container' per se, but it _is_ somewhat similar to `std::filesystem::path` maybe. 
Either way it feels a bit weird to have some types up there and some other types down here.